src/components/layout/footer.js

```diff
@@ -327,13 +362,179 @@ const query = graphql`
     }
 `
 
-const Footer = ({ type, is_ppc, is_ppc_redirect }) => {
+const Footer = ({ type, is_ppc_redirect }) => {
     const image_query = useStaticQuery(query)
     const { show_cookie_banner } = React.useContext(LocationContext)
 
     mobile_accordion_header_about.borderTop = 'none'
     const current_year = new Date().getFullYear()
 
+    const about_links = {
+        title:'ABOUT',
```

Comment:
```suggestion
        title: localize('ABOUT'),
```
Should this be localized?